examples/4337-gas-metering/gelato/README.md

```diff
@@ -10,35 +10,26 @@ npm run gelato:account:1balance
 
 Signer Extracted from Private Key.
 
-Init Code Created.
-
-Counterfactual Sender Address Created: 0x8Af37a939fBEd9ac9AdB04270EF28DC844256CB3
-Address Link: https://sepolia.etherscan.io/address/0x8Af37a939fBEd9ac9AdB04270EF28DC844256CB3
-
-Deploying a new Safe and executing calldata passed with it (if any).
+Appropriate calldata created.
```

Comment:
Are all logs in the `README.md` needed?

Its better to keep only required information e.g. txhash, gas usage and remove other text. Perhaps, a table can be used if data is meant to be compared here.